sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/GroupIntoBatchesTest.java

```diff
@@ -267,20 +225,9 @@ public void testWithShardedKeyInGlobalWindow() {
     PAssert.that("Incorrect batch size in one or more elements", collection)
```

Comment:
We should move the comment just above into `checkBatchSizes`:
```
    // Since with default sharding, the number of subshards of a key is nondeterministic, create
    // a large number of input elements and a small batch size and check there is no batch larger
    // than the specified size.
```